keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -154,10 +154,15 @@ Kubernetes already has a good coverage of node `No-Execute` eviction. We will ad
 - Verify the ability to enable and disable the default `TaintEvictionController`.
 - Verify the new `TaintEvictionController`to act on node taints properly.
 
+- `test/integration/node/lifecycle_test.go#TestEvictionForNoExecuteTaintAddedByUser`: [k8s-triage](https://storage.googleapis.com/k8s-triage/index.html?test=TestEvictionForNoExecuteTaintAddedByUser)
+- `test/integration/node/lifecycle_test.go#TestTaintBasedEvictions`: [k8s-triage](https://storage.googleapis.com/k8s-triage/index.html?sig=scheduling&test=TestTaintBasedEvictions)
+
 ##### E2E tests
 - Verify the new controllers to pass the existing E2E and conformance tests using the split `TaintEvictionController`.
 - Manually test rollback and the `upgrade->downgrade->upgrade` path to verify it can pass the existing e2e tests.
 
+- `test/e2e/node/taints.go`: [k8s-triage](https://storage.googleapis.com/k8s-triage/index.html?sig=scheduling&test=NoExecuteTaintManager)
+
```

Comment:
Can't comment on unchanged line - but once comment from the PRR:

> The upgrade will be tested in the planned unit and e2e tests. The rollback and upgrade-downgrade-upgrade path will be tested manually (see [Test Plan](https://github.com/kubernetes/enhancements/blob/930064e30b38f6984a9ec6edc0434b5e6c4f4449/keps/sig-scheduling/3902-decoupled-taint-manager/README.md#test-plan) section).

Were the test run? Can you please describe the scenario & setup that was run and findings (if any)?

Reply:
@wojtek-t @yuanchen8911 `Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?` is updated. please review it again. thanks.